e2e/config.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     addHearingBookingDetails: 'Add hearing details',
     createNoticeOfProceedings: 'Create notice of proceedings',
     addStatementOfService: 'Add statement of service (c9)',
+    c2Upload: 'Upload a C2',
```

Comment:
This should probably have a different name, something along the lines of 'Make a change request (C2)' or 'Change existing proceedings' - would need confirmation with design, also limited by 30 characters

Reply:
@joez-f UploadC2Documents?

Reply:
Checked with Mia and Gail - Upload a C2 was agreed on